internal/testing/testdata/testdata.go

```diff
@@ -264,6 +335,76 @@ var (
 		},
 	}
 
+	slsa1time, _ = time.Parse(time.RFC3339, "2023-01-01T12:34:56Z")
+	SlsaPreds1   = assembler.IngestPredicates{
+		IsOccurrence: []assembler.IsOccurrenceIngest{
+			{
+				Src: &model.SourceInputSpec{
+					Type:      "git",
+					Namespace: "github.com/octocat/hello-world@refs/heads",
+					Name:      "main",
+				},
+				Artifact: &model.ArtifactInputSpec{
+					Algorithm: "gitCommit",
+					Digest:    "c27d339ee6075c1f744c5d4b200f7901aad2c369",
+				},
+				IsOccurrence: &slsaIsOccurrence,
+			},
+			{
+				Pkg: &model.PkgInputSpec{
+					Type:      "guac",
+					Namespace: ptrfrom.String("generic"),
+					Name:      "_",
+					Version:   ptrfrom.String(""),
+					Subpath:   ptrfrom.String(""),
+				},
```

Comment:
Is this the expected PkgInput for
```json
            "name": "_",
            "digest": {
                "sha256": "fe4fe40ac7250263c5dbe1cf3138912f3f416140aa248637a60d65fe22c47da4"
            }
```
?

Reply:
Another thing that complicates this is name is completely optional. It can't really be relied on.